server/db/models/user.js

```diff
@@ -1,6 +1,6 @@
-const crypto = require('crypto')
-const Sequelize = require('sequelize')
-const db = require('../db')
+const crypto = require('crypto');
```

Comment:
@JoshuaSkootsky can you make your commit messages for specific

I.e. rather than 'no major change', say' prettify' or 're formatted'